pypeit/images/combineimage.py

```diff
@@ -194,6 +197,19 @@ def run(self, ignore_saturation=False, maxiters=5):
         else:
             comb_texp = exptime[0]
 
+        # Check that all spatial flexure values are consistent
+        comb_spat_flex = None
+        # remove nan (None) values. Since spat_flex is a float array,
+        # if rawImage.spat_flexure is None, it will be converted to nan
+        no_nan = np.logical_not(np.isnan(spat_flex))
+        if np.sum(no_nan) > 0:
+            if np.any(np.absolute(np.diff(spat_flex[no_nan])) > 0.1):
+                msgs.warn(f'Spatial flexure is not consistent for all images being combined: {spat_flex}.')
+                comb_spat_flex = np.round(np.mean(spat_flex[no_nan]),3)
```

Comment:
is there a preference to rounding to the nearest pixel instead of staying a `float` ?

Reply:
It is only rounding to the 3rd decimal. It's still a float. But the mean calculation can make it to be a very long number.